package.json

```diff
@@ -52,5 +58,27 @@
 	"optionalDependencies": {
 		"@sveltejs/adapter-cloudflare": "^4.2.1"
 	},
-	"type": "module"
+	"type": "module",
+	"main": "./electron/main.js",
+	"build": {
+		"appId": "com.hollama.app",
+		"productName": "Hollama",
+		"files": [
+			"build/**/*",
+			"electron/**/*"
+		],
+		"mac": {
+			"category": "public.app-category.developer-tools",
+			"target": "dmg",
+			"icon": "electron/resources/icons/mac/icon.icns"
+		},
+		"win": {
+			"target": "nsis",
+			"icon": "electron/resources/icons/win/icon.ico"
+		},
+		"linux": {
+			"target": "AppImage",
+			"icon": "electron/resources/icons/png/1024x1024.png"
+		}
+	}
```

Comment:
Didn't really make changes to your config other than specifying the paths to the icons.